epoxy-processor/src/main/java/com/airbnb/epoxy/EpoxyProcessor.java

```diff
@@ -245,12 +258,29 @@ private void updateClassesForInheritance(
 
       for (Entry<TypeElement, ClassToGenerateInfo> otherEntry : otherClasses.entrySet()) {
         if (isSubtype(entry.getKey().asType(), otherEntry.getKey().asType())) {
-          entry.getValue().addAttributes(otherEntry.getValue().getAttributeInfo());
+          if (belongToTheSamePackage(entry.getKey(), otherEntry.getKey())) {
+            entry.getValue().addAttributes(otherEntry.getValue().getAttributeInfo());
+          } else {
+            for (AttributeInfo attribute : otherEntry.getValue().getAttributeInfo()) {
```

Comment:
Would you prefer another PR for these fixes since this one has been already merged?